ament_cmake_export_libraries/cmake/environment/library_path.bat.in

```diff
@@ -31,4 +29,3 @@ goto:eof
     set "%~1=%_list%"
   )
 goto:eof
-
```

Comment:
The Linux shell script don't print messages in the environment hooks either. Instead that is done by the wrapping script calling the environment hooks: https://github.com/ament/ament_package/blob/master/ament_package/template/prefix_level/local_setup.sh.in#L60-L67

This PR together with the upcoming changes in ament_package will do the same thing on Windows.